windows/ci/build.ps1

```diff
@@ -1,6 +0,0 @@
-$ContainerBase = ':1803'
-
-docker build                                    `
-    -t "$env:IMAGE_NAME"                        `
-    --build-arg "BASE_IMAGE_VER=$ContainerBase" `
-    windows
```

Comment:
Hmm, are you sure this should be deleted altogether? I don't think it's replaced with anything in the actions workflow. Ditto for publish.ps1.

Reply:
Those scripts are unused, and they're not really useful right now: `windows/ci/build.ps1` simply calls `docker build`, and `windows/ci/publish.ps1` will not be needed even if we start producing Windows images on CI again (as we're authenticating with `docker/login-action@v1`).